### PR TITLE
added success dialogs back in and separated dialog services

### PR DESCRIPTION
**Pull request by mddengo**

### CODE REVIEW

File: src/app/components/devices/device.component.scss [hunk 1]
```diff
@@ -14,12 +14,13 @@ $black-text: #303132;
   }
   .save-template {
     font-family: "Avenir-Med", sans-serif;
-    color: $black-text;
+    color: #8ab9d6;
```
Comment (sandershihacker) [inline src/app/components/devices/device.component.scss]: It is better to use variables than magic numbers

File: src/app/components/devices/devicetemplate.component.scss [hunk 2]
```diff
@@ -1,6 +1,14 @@
 $black-text: #303132;
 
 .device-template {
+  .new-template {
+    font-family: "Avenir-Med", sans-serif;
+    color: #8ab9d6;
+    .button {
+      background-color: white;
+      box-shadow: none;
```
Comment (sandershihacker) [inline src/app/components/devices/devicetemplate.component.scss]: What is this? You can use md-button vs md-raised-button.

**Comment by mddengo** [comment]
fixed

File: src/app/components/locations/newlocation.component.ts [hunk 3]
```diff
@@ -27,9 +28,14 @@ export class NewLocationComponent {
   constructor(private locationService: LocationService, private route: ActivatedRoute,
               private router: Router,
               public snackBar: MdSnackBar,
+<<<<<<< HEAD
```
Comment (sandershihacker) [inline src/app/components/locations/newlocation.component.ts]: What is this? Be careful when merging.

File: src/app/components/locations/newlocation.component.ts [hunk 4]
```diff
@@ -58,21 +64,26 @@ export class NewLocationComponent {
...
-            this.dialogService
+            this.successDialogService
               .dialogPopup(SuccessDialogComponent, 'Added location: ' + this.name);
             this.locationService.notifyParent(this.parent._id);
             this.parent = null;
             this.name = "";
             this.type = "";
           },
           error => {
+<<<<<<< HEAD
```
Comment (sandershihacker) [inline src/app/components/locations/newlocation.component.ts]: Same here.

**Pull request merged**